Overview.bs

```diff
@@ -212,39 +212,129 @@ Opt-In Mechanism {#opt-in}
 
+Note: the use of <code>incremental-auto</code> may incur a CORS preflight request for the initial
+request of method negotiation, as the initial request sets a custom header.
+
 IFT Method Selection {#method-selection}
 ========================================
 
 <em>This section is general to both IFT methods.</em>
 
-When a page has indicated that a particular font should utilize IFT technology, the browser must determine which method to use. Different browsers may support different IFT methods, and different servers may support different IFT methods, so a negotation occurs as such:
+The client selects the IFT method to utilize for a font load using the following procedure:
+
+*  If the content has specified which method to use, such as via the <code>incremental-patch</code> or
+    <code>incremental-range</code> font tech keyword (see: <a href="#opt-in">opt-in mechanism</a>),
+    then that method should be used if the client supports it. If the specified method is not supported
+    then the client should fallback to to loading the font non-incrementally.
+
+*  Otherwise if the client does not know which particular IFT method(s) that the server hosting the
+    font supports, such as when the <code>incremental-auto</code> font tech keyword is used, then
+    <a href="#method-negotiation">method negotiation</a> should be used to determine the method.
 
-1. The browser makes the first request to the server. If the client prefers the <a href="#patch-incxfer">patch-subset method</a>, it sends the relevant [=query parameter=]. If the client prefers the range-request method, it does not send the query parameter.
-2. If the server receives the patch-subset query parameter and wishes to honor it, the server must reply with a valid <a href="#PatchResponse">patch subset response</a> which includes the <a href="#handling-patch-response">patch-subset magic number</a>. Otherwise, the server must reply with the [[RFC9110#range.requests]] <a href="https://www.rfc-editor.org/rfc/rfc9110.html#name-accept-ranges"><code>Accept-Ranges</code></a> header, if it supports HTTP Range Requests.
-3. If the client receives the patch-subset magic number, it commences using the patch-subset method. Otherwise, if the client receives the <code>Accept-Ranges: bytes</code> header, it commences using the range-request method. Otherwise, the whole font file is downloaded, and the current non-incremental loading behavior is used.
+Note: the most recently specified method by the content takes precedence. For example if on a site the
+first page specifies to use the patch subset method for a font URL, but the second page specifies
+range request for the same font URL then the second page should load the font using range request.
+Such cases will require the client to reset any previously stored state for that URL and start fresh
```

Comment:
Not sure if this Note is added to cover the hypothetical case of using two different methods on the same site and for the same font, but for sake of completeness, I guess we should also explain what should happen if the second page specifies <incremental-auto>. Possible scenarios would be to either renegotiate or to continue using previously specified <patch subset> method (which seems to be both logical and efficient from performance POV).

Reply:
Agreed we should be able to just continue with the previous method in that case. I'll add clarification to this note.